pytorch_pfn_extras/onnx/export/export.py

```diff
@@ -0,0 +1,879 @@
+import dataclasses
```

Comment:
I feel that this should be an independent python-package :)
ppe-onnx?

Reply:
Yes it could be a independent package.
Though all other `ppe.onnx` features should also be in it for easier integration test

Reply:
That could be possible, but I still think it's better to have it in PPE, because separation means:
* Forcing users to install two separate libraries
* Add extra overhead to take care of version compatibility between PPE and PPE-ONNX as they are relatively tightly-coupled

PPE has a flexible release policy, so in case we need a hot-fix inside these new modules, we can just cut a release 😄

Reply:
Btw, `pytorch_pfn_extras/onnx/export` module shadows the existing public API `pytorch_pfn_extras.onnx.export` (which is defined at `pytorch_pfn_extras.onnx.export_testcase.export`) so the direcotry needs to be renamed. Maybe `pytorch_pfn_extras/onnx/pfto_exporter` or something?